src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/JsonTest/JsonTest.cs

```diff
@@ -286,5 +296,40 @@ public async Task TestJsonReadAsync()
                 }
             }
         }
+
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.IsJsonSupported))]
+        public void TestNullJson()
+        {
+            string tableName = DataTestUtility.GetUniqueNameForSqlServer("Json_Test");
+
+            string tableCreate = "CREATE TABLE " + tableName + " (Data json)";
+            string tableInsert = "INSERT INTO " + tableName + " VALUES (@jsonData)";
+            string tableRead = "SELECT * FROM " + tableName;
+
+            using (SqlConnection connection = new SqlConnection(DataTestUtility.TCPConnectionString))
+            {
+                connection.Open();
+                using (SqlCommand command = connection.CreateCommand())
+                {
+                    //Create Table
+                    command.CommandText = tableCreate;
```

Comment:
Consider using DropTable() or CreateTable() function from DataTestUtility to ensure tables created by test are dropped properly.

Reply:
added the code for drop table